.changeset/blue-sloths-stare.md

```diff
@@ -0,0 +1,14 @@
+---
+'astro': patch
+---
+
+App class now accepts renderOptions
```

Comment:
Just checking, does this need updating in the docs here: https://docs.astro.build/en/reference/adapter-reference/#apprenderrequest-request-options-renderoptions